mypyc/test-data/run-multimodule.test

```diff
@@ -1,4 +1,14 @@
--- These test cases compile two modules at a time (native and other.py)
+-- These test cases compile two or more modules at a time.
+-- Any file prefixed with "other" is compiled.
+--
+-- Note that these are run in both regular, multi-file and separate
+-- compilation modes. See the docstrings of
```

Comment:
"both" can only refer to two things, but sounds like there are three modes here. Let's make it explicit
```suggestion
-- Note that these are run in three compilation modes: regular,
-- multi-file and separate. See the docstrings of
```